### PR TITLE
README.md: Move example content into examples/README.md

### PR DESCRIPTION
**Pull request by bkeryan**
Signed-off-by: Brad Keryan <brad.keryan@ni.com>

- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-python/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?

- Move the example content from /README.md to /examples/README.md.
- Add a note about `install_examples.py`.
- Update "note" boxes in top-level README.

### Why should this Pull Request be merged?

- Hyperlink the PyPI README to the examples directory on GitHub.
- Shorten the README that we upload to PyPI.
- Fix broken "PoetryInstallFail" image link when viewing the readme on PyPI.

### What testing has been done?

None

### CODE REVIEW

**Review by DelpireNI — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -214,7 +171,8 @@ Refer to the [Static Registration of measurements section]() for the detailed st
 
 To Statically register the examples provided, the user can copy the example directory with the service config file with the startup batch file, to the search paths and follow the [Setting up the Example Measurements](#setting-up-the-example-measurements) section to set up the measurements.
```
Comment (dixonjoel) [inline README.md:172]: This paragraph is worded awkwardly. I'd suggest
- Statically should not be capitalized.
- 'with the .serviceconfig file *and* the startup batch file
- No comma after 'the startup batch file'
- What does 'to the search paths' mean? Is that explained elsewhere?
- Doesn't this Setting up the Example Measurements link need to go to an external file now? We removed the section above.

**Review by dixonjoel — CHANGES_REQUESTED** [review]

**Comment by dixonjoel** [comment]
@bkeryan do you want this to go in before we make a release today or does it matter?

File: examples/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+- `nifgen_standard_function`: Generates a standard function waveform using an NI waveform generator.
+- `niscope_acquire_waveform`: Acquires a waveform using an NI oscilloscope.
+- `niswitch_control_relays`: Controls relays using an NI relay driver (e.g. PXI-2567).
+- `nivisa_dmm_measurement`: Performs a DMM measurement using NI-VISA and an NI Instrument Simulator v2.0.
+
+For more details about a specific example, see the `README.md` file included with the example.
+
+### Setting up the Example Measurements
+
+The example measurements shared are *poetry-based* projects. Follow the below steps to  for setting up the example measurement:
```
Comment (pbirkhol-ni) [inline examples/README.md:19]:
```suggestion
The example measurements are *poetry-based* projects. Follow the steps below for setting up the example measurement:
```

File: examples/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+1. Install `poetry`. Refer to <https://python-poetry.org/docs/#installation> for information on installing poetry.
+
+2. Open a command prompt, and change the working directory to the directory of the example measurement you want to work with.
+
+    ``` cmd
+    cd <path_of_example_measurement>
+    REM Example: cd "..\measurementlink-python\examples\dc_measurement"
+    ```
+
+3. Run `poetry install`. This command creates/updates the .venv and installs all the dependencies(including `ni-measurementlink-service` package) needed for the Example into `.venv`
```
Comment (pbirkhol-ni) [inline examples/README.md:30]:
```suggestion
3. Run `poetry install`. This command creates/updates the .venv and installs all the dependencies(including `ni-measurementlink-service` package) needed for the example into `.venv`
```

**Review by pbirkhol-ni — APPROVED** [review]

**Comment by bkeryan** [comment]
> @bkeryan do you want this to go in before we make a release today or does it matter?

Yes, I'd like to verify that the PyPI readme doesn't have any broken links.

**Review by dixonjoel — APPROVED** [review]

**Pull request merged**